client/client_test.go

```diff
@@ -203,6 +203,18 @@ func (s *clientTestSuite) TestConn_Select() {
 	require.Equal(s.T(), "test1", e)
 }
 
+func (s *clientTestSuite) TestConn_Select2() {
```

Comment:
you can start a MySQL in your local machine and run this test. This test case will use the DSN in line 33 to connect MySQL. In fact the test is invalid because `result.Values` has no element, please check you really test some row. 

And I think the original issue is about NULL? As least you should test NULL, and insert some rows to mixer_test_conn and check all columns

Reply:
the use cases have been completed and tested locally

```
[root@mysql003 ~]# go  test -v --host 192.168.10.44 
=== RUN   TestConnGenAttributes
--- PASS: TestConnGenAttributes (0.00s)
=== RUN   TestClientSuite
=== RUN   TestClientSuite/TestConn_Escape
=== RUN   TestClientSuite/TestConn_Insert
=== RUN   TestClientSuite/TestConn_Insert2
=== RUN   TestClientSuite/TestConn_Ping
=== RUN   TestClientSuite/TestConn_Select
=== RUN   TestClientSuite/TestConn_Select2
=== RUN   TestClientSuite/TestConn_SetCapability
=== RUN   TestClientSuite/TestConn_SetCharset
=== RUN   TestClientSuite/TestConn_TLS_Certificate
=== RUN   TestClientSuite/TestConn_TLS_Skip_Verify
=== RUN   TestClientSuite/TestConn_TLS_Verify
=== RUN   TestClientSuite/TestStmt_Delete
=== RUN   TestClientSuite/TestStmt_Insert
=== RUN   TestClientSuite/TestStmt_NULL
=== RUN   TestClientSuite/TestStmt_Select
=== RUN   TestClientSuite/TestStmt_Signed
=== RUN   TestClientSuite/TestStmt_Trans
=== RUN   TestClientSuite/TestStmt_Unsigned
--- PASS: TestClientSuite (0.39s)
    --- PASS: TestClientSuite/TestConn_Escape (0.01s)
    --- PASS: TestClientSuite/TestConn_Insert (0.01s)
    --- PASS: TestClientSuite/TestConn_Insert2 (0.00s)
    --- PASS: TestClientSuite/TestConn_Ping (0.00s)
    --- PASS: TestClientSuite/TestConn_Select (0.00s)
    --- PASS: TestClientSuite/TestConn_Select2 (0.01s)
    --- PASS: TestClientSuite/TestConn_SetCapability (0.00s)
    --- PASS: TestClientSuite/TestConn_SetCharset (0.00s)
    --- PASS: TestClientSuite/TestConn_TLS_Certificate (0.00s)
    --- PASS: TestClientSuite/TestConn_TLS_Skip_Verify (0.01s)
    --- PASS: TestClientSuite/TestConn_TLS_Verify (0.00s)
    --- PASS: TestClientSuite/TestStmt_Delete (0.00s)
    --- PASS: TestClientSuite/TestStmt_Insert (0.01s)
    --- PASS: TestClientSuite/TestStmt_NULL (0.01s)
    --- PASS: TestClientSuite/TestStmt_Select (0.00s)
    --- PASS: TestClientSuite/TestStmt_Signed (0.02s)
    --- PASS: TestClientSuite/TestStmt_Trans (0.02s)
    --- PASS: TestClientSuite/TestStmt_Unsigned (0.01s)
=== RUN   TestConnSuite
=== RUN   TestConnSuite/TestAttributes
=== RUN   TestConnSuite/TestExecuteMultiple
=== RUN   TestConnSuite/TestExecuteSelectStreaming
=== RUN   TestConnSuite/TestFieldList
--- PASS: TestConnSuite (0.25s)
    --- PASS: TestConnSuite/TestAttributes (0.00s)
    --- PASS: TestConnSuite/TestExecuteMultiple (0.01s)
    --- PASS: TestConnSuite/TestExecuteSelectStreaming (0.00s)
    --- PASS: TestConnSuite/TestFieldList (0.00s)
=== RUN   TestPoolSuite
=== RUN   TestPoolSuite/TestPool_Close
--- PASS: TestPoolSuite (0.01s)
    --- PASS: TestPoolSuite/TestPool_Close (0.01s)
PASS
ok      github.com/go-mysql-org/go-mysql/client 0.893s
```